src/sync.c

```diff
@@ -23,6 +23,7 @@
 #include "sync.h"
 
 #define PM_PARTITIONS 10
+#define MAX_PARTITIONS 14
```

Comment:
I assume that this number came from the for-loop on line 221 (PM_PARTITIONS +/- 2)? If so, maybe add a define to make that relationship clearer. Something like:
```
#define CFO_PARTITIONS 2
#define MAX_PARTITIONS (PM_PARTITIONS + CFO_PARTITIONS)
//...
for (i = -CFO_PARTITIONS * PARTITION_WIDTH; i < CFO_PARTITIONS * PARTITION_WIDTH; ++i)
```

Reply:
It's actually the maximum of these `partitions_per_band` numbers:

https://github.com/theori-io/nrsc5/blob/a40277e2686ea59a1a61d573b521c7adf8c9bfb5/src/sync.c#L173-L187